pix4d-cogito.yml

```diff
@@ -0,0 +1,5 @@
+name: cogito
+repo: https://github.com/Pix4D/cogito
+container_image: pix4d/cogito
+description: |
+Cogito updates the GitHub status of a commit during a build. Simple configuration and lightweight image.
```

Comment:
Hi! Just need some indentation here for valid yml!

```diff
-Cogito updates the GitHub status of a commit during a build. Simple configuration and lightweight image.
+  Cogito updates the GitHub status of a commit during a build. Simple configuration and lightweight image.
```

Thanks!

Reply:
it's funny because I use so much YAML daily and didn't spot this one :-D

Reply:
Fixed.

Reply:
Thanks @marco-m